### DISCUSSION TITLE
Unexpected behavior

### DISCUSSION THREAD

**Discussion by Yakabuff** [miguelgrinberg/flask-sock#91] [Q&A]
I tried copying your example (https://blog.miguelgrinberg.com/post/add-a-websocket-route-to-your-flask-2-x-application) and it just spams the page with this rendering the entire browser unresponsive. Any idea why?
![image](https://user-images.githubusercontent.com/3011758/203915258-eff26fdc-aeb3-4d20-8565-a3ce59f020d1.png)

**Comment by miguelgrinberg** [comment]
Did you make any changes to the code?
Did you try one of the examples in this repository?
